Realm/RLMClassInfo.hpp

```diff
@@ -101,6 +115,15 @@ class RLMSchemaInfo {
 
     // Look up by name, throwing if it's not present
     RLMClassInfo& operator[](NSString *name);
+    // Look up by table key, return none if its not present.
+    realm::util::Optional<RLMClassInfo&> operator[](realm::TableKey const& tableKey);
+
+    // Emplaces a locally derived object schema into RLMSchemaInfo. This is used
+    // when creating objects dynamically that are not registered in the Cocoa schema.
+    // Note: `RLMClassInfo` assumes ownership of `schema`.
+    void append_dynamic_object_schema(realm::ObjectSchema* schema,
+                                      RLMObjectSchema *objectSchema,
+                                      __unsafe_unretained RLMRealm *const target_realm);
```

Comment:
`__unsafe_unretained` here doesn't do anything and we normally exclude it from declarations as it just makes things less readable (and potentially misleading).